components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/authz/AuthorizationHandlerManager.java

```diff
@@ -129,6 +129,15 @@ private OAuth2AuthorizeRespDTO validateAuthzRequest(OAuth2AuthorizeReqDTO authzR
             return authorizeRespDTO;
         }
 
+        List<String> allowedScope = OAuthServerConfiguration.getInstance().getAllowedScopes();
```

Comment:
Let's change the variable name to `allowedScopes`

Reply:
changed in https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1468/commits/4d2bf08c421feabe46cf80c0cda56853705228d8